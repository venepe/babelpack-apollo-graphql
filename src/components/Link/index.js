import React from 'react'
import PropTypes from 'prop-types'
import { Link as RouterLink } from 'react-router-dom'


export const Link = ({ children, ...rest }) => (
  <RouterLink {...rest}>
    {children}
  </RouterLink>
)

Link.propTypes = {
  children: PropTypes.node.isRequired,
}

export default Link
