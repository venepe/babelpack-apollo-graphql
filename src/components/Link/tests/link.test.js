import React from 'react'
import { shallow } from 'enzyme'
import Link from '../index'


describe('Link Component', () => {
  it('renders with all props', () => {
    const wrapper = shallow(
      <Link to="/fake-link">Some Link</Link>,
    )
    expect(wrapper).toMatchSnapshot()
  })
})
