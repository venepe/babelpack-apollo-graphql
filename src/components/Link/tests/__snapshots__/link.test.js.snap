// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link Component renders with all props 1`] = `
<Link
  replace={false}
  to="/fake-link"
>
  Some Link
</Link>
`;
